ccm_web/client/src/App.tsx

```diff
@@ -79,6 +79,19 @@ function App (): JSX.Element {
 
   return (
     <div className='App'>
+      {
+        globals?.environment === 'development' &&
+        (
+          <div>
+            <div className={classes.swaggerLink}>
+              <Link href={`/swagger?csrfToken=${String(getCSRFToken())}`} target={'_blank'}>Swagger UI</Link>
+            </div>
+            <div style={{ position: 'fixed', right: '25px', top: '25px', zIndex: 999 }}>
+              <ResponsiveHelper/>
+            </div>
+          </div>
```

Comment:
having the swagger link  at the top, Is this change temporary?

Reply:
It can be temporary, if you insist on it.  I'd prefer to keep it at the top.  I was even thinking to change it further so it displays as…

> 🚧 𝘿𝙀𝙑𝙀𝙇𝙊𝙋𝙈𝙀𝙉𝙏 𝙈𝙊𝘿𝙀 - [Swagger UI](#)  🚧

I'd even like to have like a development banner at the top of the app's page.  (Appearing at the top of the iframe.)

As I wrote in the commit message for ba63309… 

> This change isn't strictly related to the issue, but I prefer the Swagger link to appear at the top of the page.  With the app loaded into an iframe and the iframe doesn't fit entirely within the page's display (at least, not in my browser), there's a lot of scrolling within scrolling I need to do when the link is at the bottom of the page.  It's much simpler at the top.  Plus, with the link at the top, we could also add a "DEVELOPMENT MODE" banner there, if we'd like.

That used to be a pretty common practice.  But if you hate it, I can remove it.

Reply:
Hi @lsloan, I'm willing to make changes like this. I am working on some layout related improvements and I think it would make sense for this to be part of that. For now, would you be amenable to removing this before merging?

Reply:
Okay, I'm working on it here: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/348
See `client/src/components/Layout.tsx`.

Reply:
Removed.

Reply:
🏁